src/chain.rs

```diff
@@ -159,7 +159,7 @@ impl ModularTransform for XYZtoLAB {
 
 struct ClutOnly {
     clut: Box<[f32]>,
-    grid_size: u16,
+    grid_size: u8,
```

Comment:
Was the motivation for this just to avoid the casts?

Reply:
That's the natural size for it, as read from the input. 

I hoped it would also explain to LLVM that grid_size.pow(4) can't overflow, but unfortunately that didn't have any effect.